safe_transaction_service/history/indexers/tx_processor.py

```diff
@@ -603,7 +603,7 @@ def __process_decoded_transaction(
                 SafeRelevantTransaction.objects.get_or_create(
                     ethereum_tx=ethereum_tx,
                     safe=contract_address,
-                    defaults={"timestamp": ethereum_tx.created},
+                    defaults={"timestamp": ethereum_tx.block.timestamp},
```

Comment:
```suggestion
                    defaults={"timestamp": internal_tx.timestamp},
```

Reply:
`timestamp` for `internal_tx` should be the same of the block (it's denormalized on the database for optimizing some queries). That way we save a database call to get the `block` for every `ethereum_tx`